vendor/wax.cartodb.js

```diff
@@ -3329,23 +3329,58 @@ wax.g.connector = function(options) {
 // Get a tile element from a coordinate, zoom level, and an ownerDocument.
 wax.g.connector.prototype.getTile = function(coord, zoom, ownerDocument) {
     var key = zoom + '/' + coord.x + '/' + coord.y;
+    var img = null;
     if (!this.cache[key]) {
-        var img = this.cache[key] = new Image(256, 256);
-        this.cache[key].src = this.getTileUrl(coord, zoom);
-        this.cache[key].setAttribute('gTileKey', key);
-        this.cache[key].setAttribute("style","opacity: "+this.opacity+"; filter: alpha(opacity="+(this.opacity*100)+");");
-        this.cache[key].onerror = function() { img.style.display = 'none'; };
+      this.cache[key] = new Image(256, 256);
+      img = this.cache[key];
+      this.cache[key].src = this.getTileUrl(coord, zoom);
+      this.cache[key].setAttribute('gTileKey', key);
+      this.cache[key].setAttribute("style","opacity: "+this.opacity+"; filter: alpha(opacity="+(this.opacity*100)+");");
+      this.cache[key].onerror = function() { img.style.display = 'none'; };
+      this.cache[key].setAttribute('calls', '1');
+    } else {
+      try {
+        var calls = parseInt(this.cache[key].getAttribute('calls'), 10);
+        calls++;
+        this.cache[key].setAttribute('calls', calls.toString());
+      } catch (exc) {
+        // Swallow
```

Comment:
Why do we need this empty catch?

Reply:
I'm protecting us from a bad parse. I'm not sure if `catch` is optional in Javascript but I prefer being explicit on `we are swallowing this exception because we don't want to break the code but if it fails we can go on`.

This pattern is clearly my old C# developer coming back, anyway.